compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -723,6 +723,12 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
         and it is only intended to be used in `alloc`."
     ),
 
+    rustc_attr!(
+        rustc_host, AttributeType::Normal, template!(Word), ErrorFollowing,
```

Comment:
I find the name host really confusing here, as I've been told that `HOST=true` means not-const, which goes directly against my intuition of HOST=true` meaning const, because the code is executed on the host compiler.